app/views/pages/badges.html.erb

```diff
@@ -49,12 +49,12 @@
   &lt;img src="https://d2fltix0v2e0sb.cloudfront.net/dev-badge.svg" alt="<%= current_user.name %>'s DEV Profile" height="30" width="30"&gt;
```

Comment:
Should the DEV here be replaced with `<%= community_name %>` as well? 👀

Reply:
🦅 Eagle eyes 😁 Thank you - yes it should! I pushed the update. 🎉